presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -193,7 +206,12 @@ private void loadValidPositions()
                         Page page = pageSource.getNextPage();
                         if (page != null) {
                             for (int i = 0; i < page.getPositionCount(); i++) {
-                                deletedRowsBuilder.add(new RowId(page, i));
+                                long originalTransaction = -1;
+                                int bucket = -1;
+                                long row;
+                                originalTransaction = BIGINT.getLong(page.getBlock(ORIGINAL_TRANSACTION_INDEX), i);
```

Comment:
keep declaration and assignment on the same line, same for bucket, row

Reply:
fixed.